qtfred/src/CheckedDataListModel.h

```diff
@@ -141,41 +194,139 @@ class CheckedDataListModel : public QAbstractListModel
 			return defaultFlags;
 	}
 
-	inline const SCP_unordered_set<T*>& getCheckedData() {
-		return checkeds;
+private: // private shared implementation for the following two methods
+	template<typename V>
```

Comment:
V got little to do with the outer template, it's just a workaround for the fact one can't const_cast an entire container, as is usual to avoid double implementation in these cases for single values.